src/Cache/FileStore.php

```diff
@@ -204,17 +205,18 @@ public function set(string $key, $value)
         }
 
         return $this->lock($cacheFile, LOCK_EX, function ($handle) use ($cacheKey, $cacheFile, $value) {
-            $contents = fread($handle, filesize($cacheFile) ?: 1) ?? '';
+            $size = fstat($handle)['size'];
```

Comment:
[fstat](https://www.php.net/manual/en/function.fstat.php) returns `false` on failure so this statement can lead to error. It would be better to have some checks to prevent that.

Reply:
Makes sense, I'll put ternary operator that return 1 in case of false like before